prow-jobs/pingcap/tidb/release-6.1-presubmits.yaml

```diff
@@ -56,5 +53,103 @@ presubmits:
       skip_report: false
       trigger: "(?m)^/test (?:.*? )?pull-br-integration-test(?: .*?)?$"
       rerun_command: "/test pull-br-integration-test"
-      branches:
-        - ^release-6\.1(\.\d+)?(-\d+)?(-v[\.\d]+)?$
+      branches: *branches
+    - name: pingcap/tidb/release-6.1/pull_lightning_integration_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: pull-lightning-integration-test
+      always_run: false
+      optional: true
+      skip_report: false
+      trigger: "(?m)^/test (?:.*? )?pull-lightning-integration-test(?: .*?)?$"
+      rerun_command: "/test pull-lightning-integration-test"
+      branches: *branches
```

Comment:
Please remove those lines. This branch's lightning pipeline has not been split from br, so this pipeline is not needed.